src/ast/ast.cpp

```diff
@@ -29,6 +30,20 @@ void PositionalParameter::accept(Visitor &v) {
   v.visit(*this);
 }
 
+std::string PositionalParameter::get_value(const BPFtrace &bpftrace)
```

Comment:
Don't think this method should be part of the AST - this class shouldn't know about bpftrace at all